sklearn/ensemble/tests/test_forest.py

```diff
@@ -1377,3 +1377,37 @@ def test_little_tree_with_small_max_samples(ForestClass):
 
     msg = "Tree without `max_samples` restriction should have more nodes"
     assert tree1.node_count > tree2.node_count, msg
+
+
+# check that a RandomForestRegressor never predicts numerical values outside of the range of values observed in the training set.
+def test_rf_regressor_prediction_range():
```

Comment:
It seems that this change is unrelated to the missing attribute. It should be removed from this PR.